common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1,4 +1,4 @@
-(function (define) {
+(function (define, Draggabilly) {
```

Comment:
@sarina [Draggabilly](http://draggabilly.desandro.com/) is a Javascript library that we use, and it's spelled correctly here. :)

As for the acceptance test failure, it looks like require.js is unable to successfully load this JS file. Perhaps there's invalid Javascript in the file?